ci-operator/step-registry/assisted/nutanix/setup/configure/assisted-nutanix-setup-configure-commands.sh

```diff
@@ -55,8 +55,8 @@ export NUTANIX_ENDPOINT='${NUTANIX_HOST}'
 export NUTANIX_PORT="${NUTANIX_PORT}"
 export NUTANIX_CLUSTER_NAME="${pe_name}"
 export NUTANIX_SUBNET_NAME="${subnet_name}"
-export API_VIP="${API_VIP}"
-export INGRESS_VIP="${INGRESS_VIP}"
+export API_VIPS="[{\"ip\": ${API_VIP}}]"
+export INGRESS_VIPS="[{\"ip\": ${INGRESS_VIP}}]"
```

Comment:
```suggestion
export API_VIPS="[{\"ip\": \"${API_VIP}\"}]"
export INGRESS_VIPS="[{\"ip\": \"${INGRESS_VIP}\"}]"
```

I think the IPs need to be quoted

Reply:
Nice, catch. Done